qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -420,6 +420,7 @@
     C3XGate,
     C4XGate,
     PauliEvolutionGate,
+    MCMTGate,
     ModularAdderGate,
```

Comment:
Are there other gates that might be missing here? what about `PermutationGate`, `MultiplierGate`  ?

Reply:
`MultiplierGate` already has this check but indeed `PermutationGate` and `Clifford` don't -- I'll add them 🙂

Reply:
Thanks for fixing it. This PR looks OK to me now.